app/assets/stylesheets/components/beta_banner.scss

```diff
@@ -0,0 +1,8 @@
+.govuk-phase-banner {
+  margin-bottom: 25px;
+}
+
+.beta-logo {
```

Comment:
This wasn't originally on the ie8 stylesheet.

2 questions here:
- Should this now be applied to ie8?
- Do we still need ie8? (basically does this go up to ie11)

Reply:
ie8 uses a different govuk frontend package. And unfortunately there are users out there on ie8. GA would show us exactly how many, but our user testing found a number. Without this ie8 package, the footer is in the header and all sorts of other problems like that.

The reason it was not originally in the ie8 file was an error.